client/src/components/public/calendar/EventType.vue

```diff
@@ -0,0 +1,380 @@
+            return [base, size];
+        },
+        calendarWidth() {
+            return this.innerWidth > 400 ? 390 : 300;
+        },
+        show() {
+            return this.date === '' ? false : true;
+        },
+        formatteddDate() {
+            return moment(this.date).format('dddd, MMMM DD');
+        }
+    },
+    methods: {
+        dateFormat(date) {
+            const months = [
+                'Jan',
+                'Feb',
+                'Mar',
+                'Apr',
+                'May',
+                'Jun',
+                'Jul',
+                'Aug',
+                'Sep',
+                'Oct',
+                'Nov',
+                'Dec'
+            ];
+            const [year, month] = date.split('-');
+            if (month) {
+                return months[month - 1] + ' ' + year;
+            } else {
+                return year;
+            }
+        },
+        selectTime(index) {
+            this.selectedTime = index;
+        },
+        getDay(date) {
+            let i = new Date(date).getDay(date);
+            return daysOfWeek[i];
+        },
+        onResize() {
+            this.innerWidth = window.innerWidth;
+        }
+    }
+};
+</script>
+
+<style>
+.main {
+    width: 100%;
+    height: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: flex-start;
+}
+
+.event-type-container {
+    margin-top: 50px;
+    box-shadow: 0 0 2px rgba(0, 0, 0, 0.5);
+    background-color: #fff;
+    width: 70%;
+    height: 70%;
+    max-width: 1000px;
+    max-height: 900px;
+}
+
+.event-type-container > div {
+    margin: 0 !important;
+}
+
+.event-info {
+    display: flex;
+    padding: 6px 0 6px 0;
+    margin-top: 20px;
+}
+
+.event-info ~ .event-info {
+    margin-top: 0px;
+}
+
+.event-info i {
+    margin-right: 10px;
+}
+
+.event-info-container {
+    padding: 0;
+    border-right: 2px solid rgb(243, 247, 247);
+}
+
+.calendar-container {
+    border-right: 2px solid rgb(243, 247, 247);
+    min-width: 420px;
+    margin: 0 !important;
+}
+
+.calendar-content {
+    display: flex;
+    width: 100%;
+    flex-wrap: wrap;
+    justify-content: center;
+}
+
+.calendar-container h3 {
+    margin: 50px 0 10px 0;
+    padding-left: 70px;
+    min-width: 480px;
+}
+
+.select-time-container {
+    margin: 0 auto !important;
+}
+
+.select-time-container h3 {
+    margin: 50px 0 10px 0;
+}
+
+.time-itms-container {
+    overflow-y: scroll;
+    min-width: 200px;
+    max-height: 600px;
+    width: 100%;
+    padding: 5px;
+}
+
+.select-time-item {
+    display: flex;
+    width: 100%;
+    justify-content: center;
+    border-radius: 3px;
+    border: 1px solid #281ac8;
+    color: #281ac8;
+    font-weight: bold;
+    margin: 5px 0;
+    cursor: pointer;
+}
+
+.confirm-event {
+    display: flex;
+    justify-content: space-between;
+}
+
+.select-time-btn {
+    width: 48%;
+}
+
+.selected-time-card {
+    width: 48%;
+    border: 1px solid #281ac8 !important;
```

Comment:
can you override styles without `!important` ? It relates for any !important in this file